From b35ac70bf29b4137ca30d5cb4a33ad6f7ba00059 Mon Sep 17 00:00:00 2001
From: Ryan McCabe <rmccabe@redhat.com>
Date: Mon, 8 Oct 2012 10:54:25 -0400
Subject: [PATCH] fence_virt: Fix typo in fence_virt(8) man page

Fix the description of when serial/vmchannel parameters are accepted.

Resolves: rhbz#761228
Signed-off-by: Ryan McCabe <rmccabe@redhat.com>
---
 man/fence_virt.8 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/fence_virt.8 b/man/fence_virt.8
index 57869da..14406ce 100644
--- a/man/fence_virt.8
+++ b/man/fence_virt.8
@@ -100,8 +100,8 @@ Specify if using fence_virt with older fence_xvmd installations
 which are using UUIDs.  Fence_virtd ignores this parameter.
 
 .SH SERIAL/VMCHANNEL PARAMETERS
-These parameters are used only when using fence_virt in multicast mode
-(e.g. by running fence_xvm).
+These parameters are used only when using fence_virt in serial mode
+(e.g. by running fence_virt).
 
 .TP
 .B -D
-- 
1.7.11.4

